common/customizable_localization/00_divinity_custom_loc.txt

```diff
@@ -567,56 +567,495 @@ GetActualBishopTitle = {
+		localization_key = councillor_court_chaplain_arcane_religion_county
 	}
 	text = {
 		trigger = {
 			liege.highest_held_title_tier < tier_duchy
+			religion = religion:arcane_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_arcane_religion_county_female
+	}
+	# Dragon
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:sect_of_the_dragons
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_kingdom_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = no
 		}
-		localization_key = councillor_court_chaplain_generic_county
+		localization_key = councillor_court_chaplain_dragon_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_duchy_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:sect_of_the_dragons
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_dragon_religion_county_female
+	}
+	# Druidism
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:druidism_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:druidism_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_kingdom_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:druidism_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:druidism_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_duchy_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:druidism_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:druidism_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_druidism_religion_county_female
+	}
```

Comment:
I would make it family's default for future use.

Reply:
done